### PR TITLE
Add comparison to feedforward neural network episode 4

### LINKED ISSUE #363: Compare CNN with MLP in terms of accuracy
Currently in episode 4 we start out with the statement 'CNNs work better on image data'. But it would be good to actually demonstrate that.
Currently we touch upon MLP for the CIFAR-10 problem in episode 4, but we only do it to show that it would have a lot of parameters compared to a CNN model trained on the same data. 
I suggest to include a small part that trains a simple MLP on the CIFAR-10 dataset, evaluates it, and maybe shows that we are overfitting because there are so many parameters. We can add this as an infobox 'How does this compare to a simple dense neural network?' right after we evaluated the CNN. We can add instructor notes to do it as a demo to save time.

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #363 

I played around a little bit with the number of nodes in the layers, I can improve the model a little bit to about 40% accuracy with a 100, 100 network, but the message stays the same.

### CODE REVIEW

**Comment by svenvanderburg** [comment]
@NidhiGowdra does this address the idea you raised in #303 ?

File: episodes/4-advanced-layer-types.Rmd [hunk 1]
```diff
@@ -397,6 +397,73 @@ plot_history(history, ['loss', 'val_loss'])
 
 It seems that the model is overfitting somewhat, because the validation accuracy and loss stagnates.
 
+::: instructor
+## Comparison with a network with only dense layers
+The callout box below compares the CNN approach with a network with only dense layers.
+You can choice how much you want to go in depth here, probably depending on time. You have several options:
```
Comment (psteinb) [inline episodes/4-advanced-layer-types.Rmd:403]:
```suggestion
Depending on time, the following discussion can be extended in depth up to your liking. You have several options:
```

**Review by psteinb — CHANGES_REQUESTED** [review]
small working improvement

File: episodes/4-advanced-layer-types.Rmd (hunk 1 above)
Comment (svenvanderburg) [reply episodes/4-advanced-layer-types.Rmd:403]: Much better!

**Comment by NidhiGowdra** [comment]
Hello @svenvanderburg, the second round went well, smooth delivery, good feedback and most of the issues previously raised have now been resolved. Appreciate the improvements and suggestions you provided to improve content delivery :)

**Comment by svenvanderburg** [comment]
Hi @NidhiGowdra that is great to hear! Please keep raising issues if you encounter any problems :)
